client/components/tinymce/plugins/wpcom-view/plugin.js

```diff
@@ -90,11 +91,13 @@ function wpview( editor ) {
 			type = $view.attr( 'data-wpview-type' );
 
 			ReactDom.render(
-				React.createElement( views.components[ type ], {
-					content: getText( view ),
-					siteId: sites.getSelectedSite() ? sites.getSelectedSite().ID : null,
-					onResize: debounce( triggerNodeChanged, 500 )
-				} ),
+				React.createElement( ReduxProvider, { store: editor.getParam( 'redux_store' ) },
+					React.createElement( views.components[ type ], {
+						content: getText( view ),
+						siteId: sites.getSelectedSite() ? sites.getSelectedSite().ID : null,
```

Comment:
I'd recommend now that we're using Redux state, we retrieve the selected site in descendent connected components via [`getSelectedSite` state selector](https://github.com/Automattic/wp-calypso/blob/f83d11bc6e3cff8eea4fed32d41461a1eb7f200e/client/state/ui/selectors.js#L1-L13).